doc/source/roadmap-detailed.rst

```diff
@@ -355,15 +355,69 @@ functions, and spheroidal wave functions. Three possible ways to handle this:
 stats
 `````
 
-This module is in good shape overall.  New functionality that's similar to
-what's already present can continue to be added; more advanced statistical
-routines may fit better in ``statsmodels``.  Some ideas for new contributions
-are:
-
-- Implementing (well-known) distributions to the ``stats.distributions``
-  framework is always welcome.
-- Continuing work on making the function signatures of ``stats`` and
-  ``stats.mstats`` more consistent, and adding tests to ensure that that
+The ``scipy.stats`` subpackage aims to provide fundamental statistical
+methods as might be covered in standard statistics texts such as Johnson's
+"Miller & Freund's Probability and Statistics for Engineers", Sokal & Rohlf's
+"Biometry", or Zar's "Biostatistical Analysis".  It does not seek to duplicate
+the advanced functionality of downstream packages (e.g. StatsModels,
+LinearModels, PyMC3); instead, it can provide a solid foundation on which
+they can build.  (Note that these are rough guidelines, not strict rules.
+"Advanced" is an ill-defined and subjective term, and "advanced" methods
+may also be included in SciPy, especially if no other widely used and
+well-supported package covers the topic.  Also note that *some* duplication
+with downstream projects is inevitable and not necessarily a bad thing.)
+
+The following improvements will help SciPy better serve this role.
+
+- Add fundamental and widely used hypothesis tests:
+
+  - Alexander-Govern test
+  - Somers' D
+  - Kendall's tau-c
+  - Page's L-test
+  - Tukey-Kramer test
+  - the various types of analysis of variance (ANOVA):
+
+    - fixed-effects and random-effects ANOVA
+    - two-way ANOVA (single replicate, uniform number of replicates, variable
+      number of replicates)
+    - multiway ANOVA (i.e. generalize two-way ANOVA)
+    - nested ANOVA
+    - analysis of covariance (ANCOVA)
+
+- Where appropriate, provide more than just a p-value as the result of a
+  statistical test; include also confidence intervals and measures of effect
+  size in the results.
+- Enhance the `fit` method of the continuous probability distributions:
+
+  - Expand the options for fitting to include:
+
+    - method of moments
+    - maximal product spacings
+    - method of L-moments / probability weighted moments
+
+  - Include measures of goodness-of-fit in the results
+  - Handle censored data
```

Comment:
+1 that seems like a clear gap

Reply:
`statsmodels` has some of this: https://www.statsmodels.org/stable/duration.html; also there are at least a dozen packages on PyPI related to Survival Analysis.  The change suggested here is fairly low level; it will allow the `fit` method (using MLE) of any continuous distribution to accept censored data.  How much of classical "survival analysis" should we eventually include in SciPy?  That's another discussion about that fuzzy line between completeness and duplication.

Reply:
For censored data there are multiple types (c.f. https://en.wikipedia.org/wiki/Censoring_(statistics)#Types ) however right censoring w/a fixed censor date/time is the most common. I agree we don't need to enumerate all things here but wanted to call out for design to account for multiple types. There is also truncation (c.f. https://en.wikipedia.org/wiki/Truncation_(statistics) ) which is not the same thing but also impacts how the likelihood would be written if censoring present. Again, don't need to enaumerate but wanted to call out for design considerations. Let me know if I can help by pitching in on design or implementation of any of the items-happy to help.

Reply:
In the current proposal, we plan to handle left-, right- and interval-censored data.  Other types would also be nice to have eventually.

Reply:
> Again, don't need to enaumerate but wanted to call out for design considerations. Let me know if I can help by pitching in on design or implementation of any of the items-happy to help.

Thanks!  Even to handle just the cases of left-, right- and interval-censored data is an interesting API design problem.  There are *many* ways to represent such data. What will be the friendliest representation for users?  When we get around to designing and implementing this, all help will be welcome!

Reply:
I'm going to hold off on expanding this beyond what I've already added.  Without digging a bit more into the subject, I'm not sure what should be in scope for SciPy at the moment.